app/src/main/java/com/duckduckgo/app/browser/BrowserActivity.kt

```diff
@@ -577,6 +576,7 @@ open class BrowserActivity : DuckDuckGoActivity(), CoroutineScope by MainScope()
         TextAlertDialogBuilder(this)
             .setTitle(R.string.appEnjoymentDialogTitle)
             .setMessage(R.string.appEnjoymentDialogMessage)
+            .setCancellable(true)
```

Comment:
This covers 1 or the 3 cases where you'll need to mark as `cancelable` 

- ✅ `showAppEnjoymentDialog`
- ❌ `showAppRatingDialog`
- ❌ `showGiveFeedbackDialog`

These will also require their corresponding view model functions to be invoked when cancelled, as you've done for `showAppEnjoymentDialog`